### PR TITLE
test(builder): add some integrations test to the datasource builder

### PR DESCRIPTION
**Pull request by Thenkei**
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

**Comment by Thenkei** [comment]
Task linked: [CU-860t03q57 Story #1.2 - Update introspection & builder - 8 pts](https://app.clickup.com/t/860t03q57)

File: packages/datasource-elasticsearch/test/helpers/elastic-search-template-manager.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+export async function deleteElasticsearchTemplate(template: string, indexPattern: string) {
+  const client = new Client({ node: ELASTICSEARCH_URL });
+
+  // Delete Template
+  await client.indices.deleteTemplate({
+    name: template,
+  });
+
+  // Delete Indices
+  const indicesResponse = await client.cat.indices({
```
Comment (realSpok) [inline packages/datasource-elasticsearch/test/helpers/elastic-search-template-manager.ts:42]: you could call my help to delete indexes. But this is not much saved code :)

**Review by realSpok — APPROVED** [review]
-ℹ️ Code review - small suggestion
  - ✅  Use pure functions when possible
  - ✅  Performance concerns
  - ✅  Security concerns
  - ✅  The PR tackle one subject only
- ✅ Automatic tests
  - ⤬  Unit tests
  - ✅  Integration tests
- ⤬ Manual tests
  -  ⤬  Test the functionality
  -  ⤬  Test error cases
- ✅  PR title
- ✅  Destination branch
- ✅  PR linked to the clickup task

File: packages/datasource-elasticsearch/test/helpers/elastic-search-template-manager.ts (hunk 1 above)
Comment (Thenkei) [reply packages/datasource-elasticsearch/test/helpers/elastic-search-template-manager.ts:42]: It doesn't work the same way. We could have multiple indexes

**Pull request merged**